monitor/data_monitoring.py

```diff
@@ -169,7 +169,7 @@ def _send_alerts(self, days_back: int):
         if alerts:
             self._send_alerts_to_slack(alerts)
 
-    def run(self, days_back: int, dbt_full_refresh: bool = False) -> bool:
+    def run(self, days_back: int, dbt_full_refresh: bool = False, vars: Optional[dict] = None) -> bool:
```

Comment:
what do you think about renaming `vars` to `dbt_vars` to avoid the shadowed name with the global `vars`, and also to remain consistent with `dbt_full_refresh`?

Reply:
agree